### PR TITLE
FDS-75 - use node v12.20.0 (LTS)

### PR DESCRIPTION
**Pull request by lordpixel**
Hi team,

Nodejs has been set to v12 for the monorepo as well as any child project/config that also enforces a specific version.

Smoke tests were performed to all commands for the monorepo such as:
`
$yarn install
`
`
$yarn cascara build
`
`
$yarn docs start
`
`
$yarn docs build
`
`
$yarn cosmos
`
`
$yarn cosmos:build
`
`
$yarn test
`
`
$yarn lint
`
`
$yarn tokens
`

Could you please double check everything works OK in case I've had missed something?

Thanks

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ export const getRollupConfig = ({ pwd, babelConfigFile }) => {
   const esConfig = {
     input,
     output: {
-      dir: `${SOURCE_DIR}/${pkgConfig?.module?.replace('/index.js', '')}`,
+      dir: `${SOURCE_DIR}/${pkgConfig.module.replace('/index.js', '')}`,
```
Comment (brianespinosa) [inline rollup.config.js:87]: @lordpixel was there a reason we needed to remove the optional chaining in Node 12 here? I think this syntax was used for packages that might not necessarily have a `module` parameter declared.

**Review by brianespinosa — CHANGES_REQUESTED** [review]
One change and a question. But I do not think these should be major. This should be ready to merge pretty quick.

File: docs/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@espressive/cascara": "file:../packages/cascara",
-    "@espressive/legacy-css": "latest",
+    "@espressive/legacy-css": "2.0.1",
```
[the remark this answers is not in the document]
Comment (lordpixel) [reply docs/package.json:12]: This kills the build, that's why I used the specific version.

File: rollup.config.js (hunk 1 above)
Comment (lordpixel) [reply rollup.config.js:87]: The elvis op. sees not to be supported. Rollup didn't like it.

**Pull request merged**